mlrun/model_monitoring/controller_handler.py

```diff
@@ -14,19 +14,19 @@
 #
 
 import mlrun
-from mlrun.model_monitoring.batch_application import BatchApplicationProcessor
+from mlrun.model_monitoring.controller import MonitoringApplicationProcessor
```

Comment:
since we decided on "Controller", let's change processor to controller (ex in MonitoringApplicationProcessor)

Reply:
https://github.com/mlrun/mlrun/pull/4624/commits/c18f2328dc64ac10063eaf59b6ffeb7968bb7743